app/searchcriteria.py

```diff
@@ -84,6 +82,19 @@ def retrieve_job_roles(job_roles, previous_jobrole_selected):
     return add_job_roles
 
 
+def retreiveIATstatuses():
```

Comment:
Ditto, could use renaming.

Reply:
If this only returns static data, should this be a function?

Reply:
Perhaps. Originally it was to be  dynamically allocated and get all distinct values from the database, however I realised then you wouldn't be  able to see all the options if someone wasn't assigned it. It  might also be unneeded stress on the database.